src/CMakeLists.txt

```diff
@@ -7,4 +7,7 @@ endif()
 
 target_link_libraries(${PROJECT_NAME} ${CCD_LIBRARIES} ${OCTOMAP_LIBRARIES} ${Boost_LIBRARIES})
 
-install(TARGETS ${PROJECT_NAME} ARCHIVE DESTINATION lib/ LIBRARY DESTINATION lib/)
+install(TARGETS ${PROJECT_NAME}
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Comment:
No worries.  It's more important for the case where you are shipping a mix of dlls and exes linked to them.  Specifying that the dlls belong in the bindir makes it so that they are in the same directory as executables, which lets you run the executables without having to edit your path environment variable (the current directory is the windows' first library search path.)